tools/rna_tools/paralyzer/paralyzer.xml

```diff
@@ -0,0 +1,380 @@
+<tool id="paralyzer" name="PARalyzer" version="0.1.0">
+
+    <description>A method to map interaction sites between RNA-binding proteins
+      and their targets</description>
+
+    <requirements>
+        <!-- conda dependency -->
+        <requirement type="package" version="1.5">paralyzer</requirement>
+        <requirement type="package" version="324">ucsc-fatotwobit</requirement>
+    </requirements>
+
+    <command>
+<![CDATA[
+    ## genome source
+    #if $refGenomeSource.genomeSource == "history":
+        faToTwoBit '$refGenomeSource.ownFile' ownFile.2bit &&
+        echo 'GENOME_2BIT_FILE=ownFile.2bit' > input.ini &&
+    #else
+        echo 'GENOME_2BIT_FILE=$refGenomeSource.builtin.fields.path' > input.ini &&
+    #end if
+
+    echo 'SAM_FILE=$input_sam$collapse' >> input.ini &&
+
+    #if $methods.choice == "ADDITIONAL_NUCLEOTIDES_BEYOND_SIGNAL":
+      echo 'ADDITIONAL_NUCLEOTIDES_BEYOND_SIGNAL=$methods.max_num' >> input.ini &&
```

Comment:
Do you know the config file feature of Galaxy? You should really use this, it is so much simpler and cleaner.
Look at this one: https://github.com/galaxyproject/tools-iuc/blob/596a23abc0c657eff218d16df685b427ae7cdd0a/tools/vegan/vegan_fisher_alpha/vegan_fisher_alpha.xml#L16